OsmAnd/src/net/osmand/plus/views/Zoom.java

```diff
@@ -46,6 +46,23 @@ public void zoomOut() {
 		changeZoom(-1);
 	}
 
+	public void smallZoomIn() {
+		zoomFloatPart += 0.01f;
```

Comment:
what this constants about, looks really wrong. We need to adjust exactly proper zoom up to 0.1 not to 0.01 delta